### PR TITLE
strip the model and improve output of softmax layer

### LINKED ISSUE #46: Strip the model...
Strip activation functions and loss function

### PR DESCRIPTION
**Pull request by Bartzi**
fixes #46

### CODE REVIEW

**Review by Jopyth — APPROVED** [review]
:+1:
